sites/wordpress/sidebar-content.html

```diff
@@ -0,0 +1,146 @@
+<!-- MUGAR - BASEMENT -->
+
+<h3>Services</h3>
+<ul>
+  <li>Film Reading and Digitization Machines</li>
+  <li>Help Desk</li>
+  <li>Printer</li>
```

Comment:
we likely don't need all of this content, since...
- there's a staging and prod 
- wordpress is sufficiently stable to not be concerned 
- it would be very annoying to have to keep this updated as well.